docs/_static/custom-icon.js

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************************
+ * Set a custom icon for pypi as it's not available in the fa built-in brands
+ */
+FontAwesome.library.add(
+  (faListOldStyle = {
+    prefix: "fa-custom",
+    iconName: "pypi",
+    icon: [
+      17.413, // viewBox width
+      20.139, // viewBox height
+      [], // ligature
+      "e001", // unicode codepoint - private use area
+      "m16.914 4.355-3.04 1.107v3.912l3.339-1.215v-3.913zm0.299 7.839v-3.913l-3.336 1.215v3.913zm-3.45 1.253v-3.91l-3.3 1.2v3.913zm-3.436 5.286v-3.912l-3.313 1.206v3.912zm0.136-3.939v3.868l3.314-1.206v-3.869l-3.314 1.206zm2.093 1.882c-0.367 0.134-0.663-0.074-0.663-0.463s0.296-0.814 0.663-0.947c0.365-0.133 0.662 0.075 0.662 0.464s-0.297 0.814-0.662 0.946zm-6.038-8.624 0.365-0.132-3.285-1.197-3.323 1.21 0.102 0.037 3.184 1.16zm7.282 1.349v-3.913l-3.337 1.214v3.913zm-3.449 1.254v-3.913l-3.302 1.202v3.912zm-3.415-2.672-3.336 1.215v3.913l3.336-1.215zm-6.736 3.919 3.322 1.209v-3.913l-3.322-1.209zm3.433-5.292 3.281 1.193v-3.868l-3.28-1.193zm10.167-5.158-3.317 1.207v3.913l3.317-1.207zm-3.557 1.188-3.226-1.173-3.269 1.19 3.225 1.174zm-1.585 0.703-1.624 0.592v3.868l3.317-1.207v-3.868zm-0.391 2.778c-0.367 0.134-0.662-0.074-0.662-0.464s0.295-0.813 0.662-0.946c0.366-0.133 0.663 0.074 0.663 0.464s-0.297 0.813-0.663 0.946zm-4.745 11.026v-3.914l-3.322-1.209v3.913zm3.449-1.228v-3.913l-3.371 1.227v3.913zm0.078-0.028 3.3-1.2v-3.914l-3.3 1.2zm10.035-11.72-3.21-1.17v2.338zm-3.404-2.791-3.24-1.18-3.27 1.19 3.247 1.182z", // svg path (https://simpleicons.org/icons/pypi.svg)
```

Comment:
I noticed the logo was a bit wrong (I had accidentally cut out one cube face). This fixes it but also restores the light/dark snakes.  If that turns out to look bad I'll revert to all-dark snakes (but with the missing cube face fixed).

```suggestion
      17.313, // viewBox width
      19.807, // viewBox height
      [], // ligature
      "e001", // unicode codepoint - private use area
      "m10.378 0.2-3.2376 1.1764 3.1869 1.1609 3.2376-1.1792zm-3.4137 1.2405-3.2348 1.1764 3.1841 1.1609 3.2355-1.1764zm6.7147 0.0028-3.2933 1.2004v3.8237l3.2933-1.1983zm-3.413 1.2433-3.2912 1.1976v3.8237l0.17541-0.0641 3.1158-1.1342zm-6.6535 0.01831v3.823l3.2426 1.1799v-3.8237zm10.186 0.08242v0.12328l3.1763 1.1574-1.0341 0.37688-2.1802 0.7932-0.0014 0.12962 0.03945-0.01409v7.21e-4l1.5251-0.55651 1.766-0.64316v3.8237l-1.809 0.65936-1.4821 0.5396v7.22e-4l-0.11976 0.16977v3.8251l-1.6463 0.59807v-7.22e-4l-1.647 0.59878v-3.8202l-0.03734 0.01338-0.08242 0.03029v3.8209l-3.2912 1.1997-0.11976 0.16977v3.7808l-3.2425-1.1792v-1.4202l-0.09087 0.03311-0.02888-0.02043v1.4497l0.03945 0.05635 3.363 1.2243 0.01973-0.01479 0.02043 0.01479 3.413-1.2433 3.4109-1.2405 0.03945-0.05847v-3.9097l3.3736-1.2264 0.04015-0.05565v-7.9053l-0.04015-0.02677v-0.02888l-3.363-1.225zm-5.3812 1.8259a0.56569 0.80788 35 0 1 0.42619 0.49804 0.56569 0.80788 35 0 1-0.65513 0.9362 0.56569 0.80788 35 0 1-0.65584-0.45859 0.56569 0.80788 35 0 1 0.65584-0.9362 0.56569 0.80788 35 0 1 0.22894-0.039449zm5.2615 0.78334-3.2933 1.1983v3.823l3.2933-1.1976zm-10.128 1.2384-3.2348 1.1764 3.1869 1.1609 3.2348-1.1764zm6.7147 0.0021-3.2912 1.2011v3.823l3.2912-1.2004zm-3.4109 1.2433-3.2933 1.1983v3.8237l3.2933-1.1983zm-6.6556 0.01619v3.8258l3.2426 1.1799v-3.8237zm16.89 0.21133v3.8223l-3.2912 1.1983v-3.823l2.6903-0.97918zm-10.235 3.7237-3.2933 1.1983v3.8237l3.2933-1.1983zm-6.6556 0.01902v3.823l3.2426 1.1799v-3.8237zm13.48 1.4483v3.8258l-3.2933 1.1983v-3.8237zm-3.413 1.2433v3.823l-3.2912 1.1983v-3.823zm2.4282 0.37899c-0.07325-0.0025-0.15078 0.01096-0.22894 0.03945-0.36199 0.13187-0.65533 0.55105-0.65513 0.9362 1.74e-4 0.38488 0.2934 0.59014 0.65513 0.45859 0.36199-0.13187 0.65533-0.55105 0.65513-0.9362-7.8e-5 -0.291-0.16997-0.48954-0.42619-0.49804z", // svg path (https://simpleicons.org/icons/pypi.svg)
```